### DISCUSSION TITLE
Can’t access to window.solana from Chrome extension

### DISCUSSION THREAD

**Discussion by scalprox** [phantom/docs#165] [Help & Support] [answered]
Hi,
In my extension I need to auth the client with a signed transaction. But
I'm trying to access to the object window.solana from the content script of my Chrome extension and I'm not able to read that, it's undefined I supposed it's due to isolated script execution ? Or am I missing something
Thanks !

**Comment by bfriel** [comment, marked as answer]
Hi @scalprox , Phantom currently does not inject its provider into other chrome extensions. We currently only accept requests if a site is using `https://`, on `localhost`, or is `127.0.0.1`. More info: https://docs.phantom.app/solana/detecting-the-provider

**Reply by scalprox** [reply]
Ok thanks for the response, so what are my possibilities if I need to make the user sign a transaction to proof of ownership ?

**Reply by Saif0007** [reply]
> Ok thanks for the response, so what are my possibilities if I need to make the user sign a transaction to proof of ownership ?

@scalprox did you find any solution for this?

**Reply by scalprox** [reply]
> > Ok thanks for the response, so what are my possibilities if I need to make the user sign a transaction to proof of ownership ?
> 
> 
> 
> @scalprox did you find any solution for this?

Not really..
In the extension there's nothing you can do. But I created a little website to handle the signature process.
